### PR TITLE
add migration for auto reboot timeout value

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: anupritaisno1 <www.anuprita804@gmail.com>
Change-Id: I3a13154da73f2259bfa3d48197035b76240794a6
Signed-off-by: anupritaisno1 <www.anuprita804@gmail.com>

### CODE REVIEW

**Comment by thestinger** [comment]
@anupritaisno1 Needs to be rebased because I dropped 2 commits.

**Comment by ghost** [comment]
@thestinger it's updated

File: packages/SettingsProvider/src/com/android/providers/settings/SettingsProvider.java [hunk 1]
```diff
@@ -4882,6 +4882,16 @@ private int onUpgradeLocked(int userId, int oldVersion, int newVersion) {
...
                 }
 
+                if (currentVersion == 191) {
+                        // Version 191: Update auto reboot constant string for GrapheneOS
+                        public static final String OLD_TIMEOUT_STRING = "settings_reboot_after_timeout";
+                        final long timeoutVal = Settings.Global.getLong(mContext.getContentResolver(), OLD_TIMEOUT_STRING, 0);
+                        if (timeoutVal != 0) {
+                                Settings.Global.putLong(mContext.getContentResolver(), Settings.Global.SETTINGS_REBOOT_AFTER_TIMEOUT, timeoutVal);
+                        }
+                        currentVersion = 192;
```
Comment (thestinger) [inline packages/SettingsProvider/src/com/android/providers/settings/SettingsProvider.java]: I don't really want to change this ourselves. It's going to conflict with upstream changes.

Comment (ghost) [reply packages/SettingsProvider/src/com/android/providers/settings/SettingsProvider.java]: fixed

**Comment by ghost** [comment]
updated it to fix the issue

**Comment by ghost** [comment]
This has been merged

**Pull request closed without merging**